doc/shells.rst

```diff
@@ -55,6 +55,13 @@ PuDB supports the following external shells.
 - `bpython <https://bpython-interpreter.org/>`_
 - `ptpython <https://github.com/jonathanslenders/ptpython>`_
 
+The `IPython` shell can also be used in a server-client fasion, which is
```

Comment:
Make this paragraph part of the enumeration above.

Reply:
Fixed in b4d73d3